bigtable-dataflow-parent/bigtable-hbase-beam/pom.xml

```diff
@@ -33,7 +33,7 @@ limitations under the License.
       <dependency>
         <groupId>com.google.errorprone</groupId>
         <artifactId>error_prone_annotations</artifactId>
-        <version>2.3.3</version>
+        <version>2.3.4</version>
```

Comment:
is this intentional? I'm a bit surprised that this would change

Reply:
The enforcer fails with:
```
+-com.google.cloud.bigtable:bigtable-hbase-beam:1.14.1-SNAPSHOT
  +-com.google.cloud.bigtable:bigtable-hbase-1.x-shaded:1.14.1-SNAPSHOT
    +-com.google.cloud.bigtable:bigtable-metrics-api:1.14.1-SNAPSHOT
      +-com.google.api:api-common:1.10.1
        +-com.google.guava:guava:29.0-android
          +-com.google.errorprone:error_prone_annotations:2.3.3 (managed) <-- com.google.errorprone:error_prone_annotations:2.3.4
```